src/apps/backend/modules/task/types.ts

```diff
@@ -63,11 +72,11 @@ export class TaskNotFoundError extends ApplicationError {
   }
 }
 
-export class TaskWithNameNotFoundError extends ApplicationError {
+export class TaskWithTitleNotFoundError extends ApplicationError {
```

Comment:
This should not be used, as title is not unique

Reply:
@aatiqJalanSWE - Why was this resolved?

I don't think `TaskWithTitleNotFoundError` should be valid any more as title is no longer unique and we should not be searching via title

Reply:
yes, fix this